backend/app/api/routes/campaigns.py

```
@@ -0,0 +1,354 @@
+async def get_brand_id_from_user(user_id: str) -> str:
+    """Get brand ID from user ID."""
+    supabase = supabase_anon
+
+    try:
+        response = supabase.table("brands").select("id").eq("user_id", user_id).single().execute()
+
+        if not response.data:
+            raise HTTPException(status_code=404, detail="Brand profile not found")
+
+        return response.data["id"]
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)